internal/pkg/aws/sessions/sessions.go

```diff
@@ -24,8 +24,9 @@ import (
 const (
 	userAgentHeader = "User-Agent"
 
-	credsTimeout  = 10 * time.Second
-	clientTimeout = 30 * time.Second
+	maxRetriesOnRecoverableFailures = 8 // Default provided by SDK is 3 which means requests are retried up to only 2 seconds.
```

Comment:
Hmm why do we need to make this change? Like which one requires this change?

Reply:
sorry the explanation is buried in the commit message here: https://github.com/aws/copilot-cli/pull/1995/commits/e4a1bb235bb21564c99472a21087eddfa03b6886 my bad

> We have multiple stacks running in parallel in our e2e test, and we
observed that IAM role deletion is a common call that can fail due to
throtlling errors from the service. This change increases the number of
times we retry in the CLI for retryable errors.